src/include/columnar/columnar_tableam.h

```diff
@@ -50,8 +50,7 @@ typedef struct ColumnarScanDescData *ColumnarScanDesc;
 
 const TableAmRoutine * GetColumnarTableAmRoutine(void);
 extern void columnar_tableam_init(void);
-extern void columnar_tableam_finish(void);
-
+extern bool CheckCitusVersion(int elevel);
```

Comment:
I couldn't follow how this works. The definition of this function is under distributed/, but we have another prototype for the same in tableam.h.

Reply:
We wanted to remove the header because we didn't want the other dependencies the distributed/ header file included to cause any build problems. The only dependency we need from the distributed header file is CheckCitusVersion so this is also cleaner.

Reply:
@onurctirtir, the header just contains a bunch of prototypes and other headers. Instead of including those headers which also declare a bunch of other things we don't need (and may introduce problems), we only want the prototype for the one remaining citus symbol that we're dependent on after this PR: `CheckCitusVersion`.

Come to think of it, it would be slightly cleaner to just include this declaration in `columnar_tableam.c` instead, because all callsites are in there. But it'll be removed later when we create a separate version of CheckCitusVersion() anyway.

Reply:
Thank you @yxu2162 & @jeff-davis :)